framework/src/problems/FEProblemBase.C

```diff
@@ -8177,6 +8152,35 @@ FEProblemBase::notifyWhenMeshChanges(MeshChangedInterface * mci)
   _notify_when_mesh_changes.push_back(mci);
 }
 
+void
+FEProblemBase::notifyWhenMeshDisplaces(MeshDisplacedInterface * mdi)
+{
+  _notify_when_mesh_displaces.push_back(mdi);
+}
+
+bool
+FEProblemBase::updateDisplacement(const ExecFlagType & exec_type)
```

Comment:
I also dont see why we pass the exec flag since the problem knows its current exec flag. I would rather we have boolean parameters to control the behavior of this routine